src/loaders/ccloudResourceLoader.test.ts

```diff
@@ -442,4 +452,116 @@ describe("CCloudResourceLoader", () => {
       };
     }
   }); // getFlinkArtifacts
+
+  describe("loadProviderRegions", () => {
+    let regionsApiStub: sinon.SinonStubbedInstance<RegionsFcpmV2Api>;
+
+    beforeEach(() => {
+      const mockSidecarHandle: sinon.SinonStubbedInstance<sidecar.SidecarHandle> =
+        sandbox.createStubInstance(sidecar.SidecarHandle);
+      regionsApiStub = sandbox.createStubInstance(RegionsFcpmV2Api);
+      mockSidecarHandle.getRegionsFcpmV2Api.returns(regionsApiStub);
+      sandbox.stub(sidecar, "getSidecar").resolves(mockSidecarHandle);
```

Comment:
We're trying to standardize some of the common stubbing patterns ([issue](https://github.com/confluentinc/vscode/issues/2233)), so let's use [`getSidecarStub()`](https://github.com/confluentinc/vscode/blob/main/tests/stubs/sidecar.ts#L9) here:
```suggestion
      stubbedSidecar = getSidecarStub(sandbox);
      regionsApiStub = sandbox.createStubInstance(RegionsFcpmV2Api);
      stubbedSidecar.getRegionsFcpmV2Api.returns(regionsApiStub);
```